README.md

```diff
@@ -85,33 +120,58 @@ If don't want to upload the files to GCS then run with the `--no-gcs` flag.
 When you are finished exit the pod with `ctrl + D` or by running `exit`. This will disconnect, then you can delete the pod with `make delete-pod`.
 
 
-## Tests and Checks
+## Rabbit Queue Message Import/Export
 
-Run the unit tests and style/safety checks with
+Two python scripts to allow us to backup and restore the contents of a Rabbit queue:
+- Dump messages from a Rabbit queue to files
+- Dump message files to a Rabbit queue
 
+### Run Locally
+Run the script to export a Rabbit queue to message files with:
 ```bash
-make test
+pipenv run python dump_queue_to_files.py <queue name> <output directory> --no-gcs
 ```
 
-## Docker
-Build the image tagged as `eu.gcr.io/census-rm-ci/rm/census-rm-qid-batch-runner:latest` with
+Run the script to export a Rabbit queue to message files with:
 ```bash
-make build
+pipenv run python dump_files_to_queue.py <queue name> <source directory> <destination directory>
 ```
 
-## Importing this code as a package
-This repo contains a setup.py so that it can be installed through pip/pipenv as a github VCS dependency
+### Run in Kubernetes
 
-### Install from master
-To install the package from master run 
+#### Prerequisites
+The generate print files script needs a GCS bucket named `<PROJECT_ID>-<queue name>-queue-dump-files` in the project GCloud pointing at and bucket get/object create permissions.
```

Comment:
Coud mention the alternative `--no-gcs` flag.

Reply:
mentioned in the running locally section